pkg/util/utils.go

```diff
@@ -139,3 +126,29 @@ func EscapeSingleQuote(s string) string {
 	r := strings.NewReplacer(`'`, `'"'"'`)
 	return r.Replace(s)
 }
+
+// IsDirEmpty checks whether directory is empty or not
+// https://stackoverflow.com/a/30708914/1120530
+func IsDirEmpty(name string) bool {
+	f, err := os.Open(name)
+	if err != nil {
+		return false
+	}
+	defer f.Close()
+
+	_, err = f.Readdirnames(1)
+	if err == io.EOF {
+		return true
+	}
+
+	return false
+}
+
+// IsExists checks whether the file/directory exists or not
+func IsExist(name string) bool {
```

Comment:
this helper already exists in file.go and I think in this case "Exists" is the better name. If you really want a prefix it should be "DoesExist".